packages/embark/src/lib/modules/geth/blockchain.js

```diff
@@ -425,24 +426,19 @@ export function BlockchainClient(userConfig, options) {
   if (options.clientName) userConfig.client = options.clientName;
   // Choose correct client instance based on clientName
   let clientClass;
-  switch (userConfig.client) {
-    case constants.blockchain.clients.geth:
-      clientClass = GethClient;
-      break;
-
-    // case constants.blockchain.clients.parity:
-      // clientClass = ParityClient;
-      // break;
-    default:
-      console.error(__('Unknown client "%s". Please use one of the following: %s', userConfig.client, Object.keys(constants.blockchain.clients).join(', ')));
-      process.exit(1);
+
+  if (communicationConfig) {
```

Comment:
It's a bit weird to just check for the presence of the config. Having an actual flag saying "it's a whisper node" would make more sense. Otherwise, one could think that it needs the communicationConfig, pass it to it, and not understand why it doesn't start the right node